core/src/kura.rs

```diff
@@ -95,7 +95,7 @@ pub trait KuraTrait:
     /// World for applying blocks which have been stored on disk
     type World: WorldTrait;
 
-    /// Default [`Kura`] constructor.
+    /// Construct [`Kura`].
```

Comment:
```suggestion
    /// Constructs `Kura`.
```

Reply:
I'd recommend this for others

Reply:
I actually prefer Sato's style here. Most style guides prefer imperative mood for function documentation. The reasoning is mostly vague, but since most of our other functions use imperative mood, I'd prefer that they remain as is.

Reply:
Sure